cmd/gateway-s3.go

```diff
@@ -85,6 +85,10 @@ func s3ToObjectError(err error, params ...string) error {
 		}
 	case "XAmzContentSHA256Mismatch":
 		err = SHA256Mismatch{}
+	case "NoSuchUpload":
```

Comment:
Can we bring in a test case for s3ToObjectError() ? @krisis

Reply:
@harshavardhana  done